plugins/nf-lamin/src/main/nextflow/lamin/api/LaminInstanceSettings.groovy

```diff
@@ -1,29 +1,26 @@
 package nextflow.lamin.api
 
 import java.util.UUID
+import groovy.transform.RecordType
 import groovy.transform.CompileStatic
 
 @CompileStatic
-record LaminInstanceSettings(
-    UUID id,
-    String owner,
-    String name,
-    UUID schemaId,
-    String apiUrl
-) {
+@RecordType
+class LaminInstanceSettings {
```

Comment:
LaminDBInstanceSettings or DBInstanceSettings or InstanceSettings ☺️ -- "Lamin instance" is not a thing